### PR TITLE
FPGA: Update README for code samples that are affected by crash

### PR DESCRIPTION
**Pull request by aejjehint**
# Existing Sample Changes
## Description

Update the readme for the code samples that may crash with the 2024.2 release.

### CODE REVIEW

**Review by yuguen-intel — APPROVED** [review]

**Comment by yuguen-intel** [comment]
Waiting for @whitepau 's approval

**Comment by whitepau** [comment]
waiting for confirmation from broader intel and marketing on how we can actually distribute the patch so the recommendation text is properly actionable.

**Review by yuguen-intel — APPROVED** [review]

**Pull request merged**
